src/components/dashBoard/OverlappingAvatars.tsx

```
@@ -6,8 +6,8 @@ export default function OverlappingAvatars({ users }) {
   return (
     <div className="flex items-center">
       <div className="flex -space-x-3">
+        <TooltipProvider>
         {users.map((user, index) => (
-          <TooltipProvider>
             <Tooltip>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)